hms-api/src/handler/comment/create.ts

```diff
@@ -0,0 +1,19 @@
+import {buildResponse} from '../../rest/responses';
+import {createComment} from '../../service/idea_comment-service';
+import {wrapHandler} from '../handler-wrapper';
+import CommentCreateRequest from '../../rest/Comment/CommentCreateRequest';
+import CommentCreateResponse from '../../rest/Comment/CommentCreateResponse';
+
+export async function create(event, context, callback) {
+  await wrapHandler(async () => {
+    const request = CommentCreateRequest.parse(event.body);
+    const comment = await createComment(
+      request.ideaId,
+      request.userId,
```

Comment:
the userID of the person that is creating it should come from the authentication token and not the request (this prevents you from creating comments as someone else)

Reply:
I bet this would then be also the case for the other create handler?

Reply:
yes, everywhere

Reply:
don't we do this already for other things?

Reply:
nope we always use request.userId everywhere

Reply:
ok cool, well something we should adjust soon